govcd/vapp_network.go

```diff
@@ -175,9 +176,13 @@ func (vapp *VApp) UpdateNetworkNatRulesAsync(networkId string, natRules []*types
 	}
 	networkToUpdate.Xmlns = types.XMLNamespaceVCloud
 
-	if networkToUpdate.Configuration.Features.NatService == nil {
+	if networkToUpdate.Configuration.Features.NatService == nil && networkToUpdate.Configuration.Features.FirewallService == nil {
 		return Task{}, fmt.Errorf("provided network isn't connected to org network or isn't fenced")
```

Comment:
How do you deduce from NAT and FW services being nil that it is "network isn't connected to org network or isn't fenced"? This asks for a comment in the code.

Reply:
don't know what to write.. will try

Reply:
added